docs/load_ldoc.cmake

```diff
@@ -1,23 +1,23 @@
 # To avoid copy pasting, some documentation is stored in reusable files
 set(SHAPE_FILE "${SOURCE_DIR}/docs/common/${SHAPE_NAME}.lua")
 
-set(path "${SOURCE_DIR}/docs/common/")
+foreach(path ${BUILD_DIR}/docs/common/;${SOURCE_DIR}/docs/common/)
```

Comment:
This is just for `rules_index.ldoc`, right? If so, when that file is generated in the build dir, then this piece of code is not needed.

(Also, I just remember that `rules_index.ldoc` is copied around... doesn't that mean that ldoc ends up parsing both the in-source and in-build copy of the file?)

Reply:
see previous comment